test-frame-log-collector/src/main/java/io/skodjob/testframe/LogCollector.java

```diff
@@ -64,7 +76,7 @@ public LogCollector(LogCollectorBuilder builder) {
      *
      * @param kubeCmdClient     {@link KubeCmdClient} - usually mocked.
      */
-    /* test */ public void setKubeCmdClient(KubeCmdClient<Kubectl> kubeCmdClient) {
+    /* test */ public void setKubeCmdClient(KubeCmdClient<?> kubeCmdClient) {
```

Comment:
Because these are in the `builder` class as well, I think we can remove them from here, right? (I mean this method and the one for KubeClient)